README.md

```diff
@@ -125,13 +125,48 @@ jobs:
     steps:
     # The checkout step
     - uses: actions/checkout@master
-    - uses: rojopolis/spellcheck-github-actions@0.23.2
+    - uses: rojopolis/spellcheck-github-actions@0.24.0
       name: Spellcheck
       with:
         source_files: README.md CHANGELOG.md notes/Notes.md
         task_name: Markdown
 ```
 
+## Specify a PySpelling Output Artifact
+
+In order to make it easier to process larger amount of output. The action allows for the user to enable the generation of an artifact.
+
+The `output_file` both enables the generation of the artifact and specifies the name of the file archived.
```

Comment:
The action `rojopolis/spellcheck-github-actions` does not upload the artifact itself, therefore the example should include the `actions/upload-artifact` step as in the changelog above. Something along the  can be introduced along the lines of

```suggestion
The optional `output_file` input parameter, if specified, defines the name of the generated file containing the spellcheck output. Such file can then be stored as workflow artifact using the `actions/upload-artifact` step.
```